src/js/remoteServices.js

```diff
@@ -1,8 +1,12 @@
 const projectRegex = /\[([\w-]+)\]/
+const projectIdRegex = /P\d{5}/gm
```

Comment:
The convention is to have the project number in brackets, e.g. `Title [P12345]`. This is too specific for your use case, better if you can stick to this convention and remove this, OK?